Frends.Kafka.Consume/Frends.Kafka.Consume.Tests/MessageProducer.cs

```diff
@@ -0,0 +1,349 @@
+﻿using Avro;
+using Avro.Generic;
+using Confluent.Kafka;
+using Confluent.SchemaRegistry;
+using Confluent.SchemaRegistry.Serdes;
+using Frends.Kafka.Consume.Definitions;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System.Globalization;
+using System.Runtime.InteropServices;
+using System.Text;
+
+#pragma warning disable CS8600,CS8603,CS8601,CS8604, CS8602
+namespace Frends.Kafka.Consume.Tests
```

Comment:
Use file-wise namespace to reduce indents overall.
So just 
`namespace XXX;`
`internal class YYY {}`